test/e2e/framework/service_util.go

```diff
@@ -535,7 +576,12 @@ func (j *ServiceTestJig) RunOrFail(namespace string, tweak func(rc *v1.Replicati
 	}
 	result, err := j.Client.Core().ReplicationControllers(namespace).Create(rc)
 	if err != nil {
-		Failf("Failed to created RC %q: %v", rc.Name, err)
+		Failf("Failed to create RC %q: %v", rc.Name, err)
+	}
+	pdb := j.newPDBTemplate(namespace)
```

Comment:
So this still modifies every service test to now use a pod disruption budget? I'm not sure that's ok.